### PR TITLE
user keymaps for ergodox_ez/dvorak_42_key / atreus/dvorak_42_key

### PR DESCRIPTION
**Pull request by lucwastiaux**
## Description

user keymap changes for ergodox_ez/dvorak_42_key / atreus/dvorak_42_key
add windows 10 workspace shortcuts, add vscode shortcuts, remove Cloud9 shortucts.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/atreus/keymaps/dvorak_42_key/keymap.c [hunk 1]
```diff
@@ -108,6 +114,17 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
             SEND_STRING(SS_LCTRL("e"));
             return true;
 			break;
+    case W10_TASKVIEW:
+        SEND_STRING(SS_LGUI(SS_TAP(X_TAB)));
+        return true;    
+        break;
+    case W10_WORKSPACE_LEFT:
+        SEND_STRING(SS_LGUI(SS_LCTRL(SS_TAP(X_LEFT))));
```
Comment (drashna) [inline keyboards/atreus/keymaps/dvorak_42_key/keymap.c:122]:
```suggestion
        tap_code16(G(C(KC_LEFT)));
```

**Comment by lucwastiaux** [comment]
Hi @drashna , I accepted your changes, would it be possible to merge this PR ? Thank you.

File: keyboards/atreus/keymaps/dvorak_42_key/keymap.c [hunk 2]
```diff
@@ -35,13 +36,18 @@ enum custom_keycodes {
...
+  W10_TASKVIEW,
+  W10_WORKSPACE_LEFT,
+  W10_WORKSPACE_RIGHT,
+
 };
 
 // building instructions:
 // make atreus:dvorak_42_key
 
 // flashing instructions:
```
Comment (fauxpark) [inline keyboards/atreus/keymaps/dvorak_42_key/keymap.c:49]:
```suggestion
```

File: keyboards/atreus/keymaps/dvorak_42_key/keymap.c [hunk 3]
```diff
@@ -35,13 +36,18 @@ enum custom_keycodes {
...
-// avrdude -p atmega32u4 -c avr109 -U flash:w:atreus_dvorak_42_key.hex  -P COM7
+// avrdude -p atmega32u4 -c avr109 -U flash:w:atreus_astar_dvorak_42_key.hex  -P COM15
```
Comment (fauxpark) [inline keyboards/atreus/keymaps/dvorak_42_key/keymap.c:50]:
```suggestion
```

`qmk flash` or `make <keyboard>:<keymap>:flash` is the preferred way to flash keymaps.

**Pull request merged**